### ISSUE TITLE
Show reviewable's friendly ID in Reviewing Area

### ISSUE THREAD

**Issue by pferreir** [indico/indico#5219]
Conference Display Interface. Right now only the titles are shown, and the friendly ID would be useful to organizers.

**Comment by mic4ael** [comment]
Is this still something you want to have implemented? :D

**Comment by pferreir** [comment]
> Is this still something you want to have implemented? :D

@thiefmaster

**Comment by ThiefMaster** [comment]
Yes. At some point for sure. Are you going through your open issues? :D

**Comment by mic4ael** [comment]
> Yes. At some point for sure. Are you going through your open issues? :D

I am preparing for Hacktoberfest :D

**Comment by noahsalvi** [comment]
It seems that `contribution.friendly_id` is already rendered before the contribution's title.

https://github.com/indico/indico/blob/7679d7972c422e8b0000b83b6ede4b7f4be6d36d/indico/modules/events/papers/templates/display/_paper_list.html#L20

<img width="1089" height="787" alt="Image" src="https://github.com/user-attachments/assets/8f58f389-a6f8-4ba5-a252-7efb00b93d2b" />

**Comment by noahsalvi** [comment]
Closed after confirming it with the issue's author.

**Comment by ThiefMaster** [comment]
Do we also show it in the paper timeline (the one you get when clicking on one of the contributions there)?

**Comment by noahsalvi** [comment]
> Do we also show it in the paper timeline (the one you get when clicking on one of the contributions there)?

Yes, it's also featured in the paper timeline view.

<img width="1049" height="971" alt="Image" src="https://github.com/user-attachments/assets/709de920-afed-4922-9551-3402eff91249" />
